apps/api/src/purchase-orders/purchase-orders.service.ts

```diff
@@ -122,9 +131,60 @@ export class PurchaseOrdersService {
         totalCost: fullDto.totalCost,
         orderedDate: fullDto.orderDate,
         expectedDeliveryDate: fullDto.expectedDeliveryDate,
+        createdBy: user.email,
+        approvers: approvers.map((x) => x.email),
```

Comment:
Adds approvers and created by emails to the payload so the email service can use them when sending emails.